packages/api/src/command/webhook/webhook.ts

```diff
@@ -80,12 +86,18 @@ export const processRequest = async (
     });
   };
 
-  const payload = webhookRequest.payload;
+  const payload = webhookRequest.payload as Record<string, string>;
   try {
+    console.log("wh key", webhookKey);
```

Comment:
this is a security issue - we shouldn't be logging keys

Reply:
sorry debug statement I forgot to remove.